readme.md

```diff
@@ -51,7 +51,7 @@ See [SECURITY.md](SECURITY.md) for more security-related information.
 
 #### 🛰 Deployment status: available on Ethereum mainnet
 
-aragonOS powers over a thousand organizations on Ethereum mainnet, securing over $200MM in funds.
+aragonOS powers over a thousand organizations on Ethereum mainnet, securing over $350M in assets.
```

Comment:
```suggestion
aragonOS powers over a thousand organizations on Ethereum mainnet, securing over $350MM in funds.
```

MM is usually used to denote millions (M = 1000 as a roman numeral) :).

I've also just aligned the text to be the same as aragon/aragon.